build.assets/Dockerfile

```diff
@@ -111,7 +111,6 @@ ENV LANGUAGE="en_US.UTF-8" \
 ARG BUILDARCH
 
 # Install packages.
-# Java JRE is required by gcloud firestore emulator.
```

Comment:
Is this comment the true? We'll see. :D

Reply:
I know that when you remove it, the Firestore emulator stops working. AFAIK nothing else requires Java, I hope so.